<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signer</class>
 <widget class="QWidget" name="signer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">&quot;border:2px groove gray;border-radius:10px;padding:2px 4px;&quot;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Arial&quot;;
color: rgb(14, 6, 255);</string>
   </property>
   <property name="text">
    <string>指针签到系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back/style/first.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>421</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">“venus–TitleBar {&quot;background-color:rgba(244,244,244,2)&quot;;border:none;}”</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>sign_in</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>123456</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>16</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 39, 42);
font: 75 9pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>账户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 39, 42);</string>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>2559926</string>
     </property>
    </widget>
    <zorder>lineEdit</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>lineEdit2</zorder>
    <zorder>pushButton</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>*表示不可为空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>26</y>
       <width>62</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 83, 114);</string>
        </property>
        <property name="text">
         <string>账户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 70, 82);</string>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 83, 114);</string>
        </property>
        <property name="text">
         <string>班级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 70, 82);</string>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>173</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
